src/Shared/MSBuildSdkTestBase.cs

```diff
@@ -47,7 +47,7 @@ public void Dispose()
 
         protected DirectoryInfo CreateFiles(string directoryName, params string[] files)
         {
-            DirectoryInfo directory = new DirectoryInfo(Path.Combine(TestRootPath, directoryName));
+            DirectoryInfo directory = Directory.CreateDirectory(Path.Combine(TestRootPath, directoryName));
```

Comment:
>Directory.CreateDirectory [](http://example.com/codeflow?start=38&length=25)

Should not create the dir here, it's important for Artifacts testing to ensure that Artifacts creates the destination dir.